beacon-chain/powchain/service.go

```diff
@@ -154,7 +158,7 @@ type Service struct {
 
 // Web3ServiceConfig defines a config struct for web3 service to use through its life cycle.
 type Web3ServiceConfig struct {
-	HTTPEndPoint       string
+	HTTPEndPoints      []string
```

Comment:
Just for the sake of consistency: sometimes it's `endpoint/Endpoint` and sometimes it's `endPoint/EndPoint`. Can we unify this?